include/deal.II/base/thread_management.h

```diff
@@ -1824,6 +1824,21 @@ namespace Threads
       return *this;
     }
 
+
+    /**
+     * Return how many tasks have been put into this group. This
+     * function does not distinguish how many of these tasks have
+     * already run and have finished, are still waiting to be
+     * scheduled to a CPU resource, or are currently running. Tasks
+     * that have been joined already are also still counted.
+     */
+    unsigned int
```

Comment:
We are never going to have billions of tasks, but I think that this will probably give us a narrowing warning. Could we just use `std::size_t` instead?

Reply:
Sure!